test/ui-test/testSuites/suite_status/tst_wallet/test.feature

```diff
@@ -120,3 +120,11 @@ Feature: Status Desktop Wallet
        Examples:
           | account_name | address                                      |
           | YugaLabs          | 0xA858DDc0445d8131daC4d1DE01f834ffcbA52Ef1   |
+
+    Scenario Outline: Can see transactions for an account
```

Comment:
could you merge this test with another one existing?

Reply:
you mean with the `add watch account` test?

Reply:
yes